whipper/program/cdparanoia.py

```diff
@@ -196,7 +196,7 @@ def getTrackQuality(self):
 
         # don't go over a 100%; we know cdparanoia reads each frame at least
         # twice
-        return min(frames * 2.0 / reads, 1.0)
+        return min(frames * 2.0 / reads, 1.0) if reads else 0
```

Comment:
If that's the reason, I think it's better to rely on the try ... except solution for that's more expressive in this case (unexpected error).